src/oop/app/src/Models/Movie.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace src\oop\app\src\Models;
+
+class Movie implements MovieInterface
+{
+
+    public string $title;
```

Comment:
You have public properties and the getters and setters to them. Maybe these properties should be protected or privat?